Packs/CloudflareWAF/ReleaseNotes/1_0_27.md

```diff
@@ -0,0 +1,10 @@
+#### Integrations
+
+
+##### Cloudflare WAF
+
+- Updated the Docker image to: *demisto/python3:3.12.8.1983910*.
+- Renamed the *The API user Token* parameter to *API Token* (no impact on existing configurations).
+- Added support for the following parameters:
```

Comment:
Please add the meaning of this. some thing like that introduce a new authentication method.